blueprints/app/files/.prettierignore

```diff
@@ -13,6 +13,7 @@
 # misc
 /coverage/
 !.*
+.*/
```

Comment:
What does this do?

Reply:
It shields dot-directories from Prettier linting, e.g. the `.git` directory. Although in hindsight I'm realizing that this will keep directories with non-JS contents like `.github` from getting linted, when in reality they probably should.

Will remove.